test/src/e2e_vm_tests/test_programs/should_pass/stdlib/assert_eq/src/main.sw

```diff
@@ -0,0 +1,76 @@
+script;
+
+use std::assert::assert_eq;
+use std::address::Address;
+use std::contract_id::ContractId;
+use std::bytes::Bytes;
+use std::vec::Vec;
+use std::identity::Identity;
+use std::b512::B512;
+
+fn main() -> bool {
+
+// test_assert_eq_u64
```

Comment:
These comments can be indented and the double `// //` can be changed to single comments